app/en/home/auth-providers/figma/page.mdx

```diff
@@ -282,13 +321,33 @@ async def get_figma_file(
 
 ## Available Scopes
 
-Figma supports various OAuth scopes that determine the level of access your application has:
+Figma supports various OAuth scopes that determine the level of access your application has. Here are the scopes used by the Arcade Figma MCP Server:
 
-- `files:read` - Read access to files
-- `file_comments:write` - Write access to file comments
+### File and Content Access
+- `file_content:read` - Read access to file structure and content (required for most file operations)
+- `file_dev_resources:read` - Read access to dev resources
 - `file_dev_resources:write` - Write access to dev resources
 - `file_variables:read` - Read access to variables
 - `file_variables:write` - Write access to variables
+
+### Library Access
+- `library_content:read` - Read access to published library content in files
+- `team_library_content:read` - Read access to team library content
+- `library_assets:read` - Read access to individual library assets (components, styles, component sets)
+
+### Comments
+- `file_comments:read` - Read access to file comments
+- `file_comments:write` - Write access to create and reply to file comments
+
+### User and Projects
+- `current_user:read` - Read access to the authenticated user's profile
+- `projects:read` - Read access to team projects and files (**Private OAuth apps only**)
+
+### Other Scopes
 - `webhooks:write` - Write access to webhooks
 
-For a complete list of available scopes and their descriptions, refer to the [Figma API documentation](https://www.figma.com/developers/api#authentication).
+<Callout type="warning">
+The `projects:read` scope is **only available in private Figma OAuth apps**. If you need to access team projects and files, you must create a private OAuth app through your Figma organization.
```

Comment:
I think we should have this warning (or link to it) next to the list of scopes.

It'd also be good to have this warning where we document the tools which depend on this scope.